src/core/lib/slice/percent_encoding.cc

```diff
@@ -72,6 +72,7 @@ const BitSet<256>& LookupTableForPercentEncodingType(PercentEncodingType type) {
     case PercentEncodingType::Compatible:
       return g_compatible_table;
   }
+  GPR_UNREACHABLE_CODE(abort());
```

Comment:
Could you add a comment on what this does please?

Reply:
Done